src_py/apiServer/transmitter.py

```diff
@@ -18,11 +19,14 @@ def __init__(self, experiment : Experiment, mainServerAddress, input_data_path :
         self.input_data_path = input_data_path
         self.mainServerAddress = mainServerAddress
         self.sourceInitAddr = self.mainServerAddress + '/sourceInit'
-        self.clientsTrainingAddress = self.mainServerAddress + '/clientsTraining'
+        self.clientsTrainingAddress = self.mainServerAddress + '/clientsTraining'  #deprecated
+        self.clientsPhaseUpdateAddress = self.mainServerAddress + '/clientsPhaseUpdate'
         self.updateCSVAddress = self.mainServerAddress + '/updateCSV'
         self.startCastingAddress = self.mainServerAddress + '/startCasting'
-        self.clientsPredictAddress = self.mainServerAddress + '/clientsPredict'
+        self.clientsPredictAddress = self.mainServerAddress + '/clientsPredict' #deprecated
         self.statisticsAddress = self.mainServerAddress + '/statistics'
+        self.terminate_address = self.mainServerAddress + '/terminate'
+        self.send_dc_json_address = self.mainServerAddress + '/sendDCJson' #TODO Guy - implement on erlang side
```

Comment:
@ohad123 @GuyPerets106 @NoaShapira8 
Change this action from sendDCJson to sendJsons 
(Since it sends both DC and Communication jsons)